Atomic/syscontainers.py

```diff
@@ -1177,7 +1177,8 @@ def _inspect_system_branch(self, repo, imagebranch):
             imgs = self._resolve_image(repo, imagebranch, allow_multiple=True)
             if imgs is None:
                 raise ValueError("Image %s not found" % imagebranch)
-            _, commit_rev = imgs[0]
+            # replace the old branch with the new branch found
+            imagebranch, commit_rev = imgs[0]
```

Comment:
Ah, the reason why I changed that image branch is because when we are deleting an unnamed image layer of an ociimage, if the user types in partial id(not the 64 characters id) into the command line, we can correctly identify the existence of an unnamed layer image, but the image id of the unnamed layer will be interpreted as the commit_rev( which can be seen in line 1180 ish in syscontainers.py in _inspect_system_branch function).
As a result, it leads to some of the information for the unnamed layer to be processed incorrectly. I am not too sure about whether this change would leave any side effects over, but fortunately I have not came across those at the current stage. So I am suggesting maybe we can leave it for now, if errors came along in the future, we can possibly change it back?